spec/index.html

```diff
@@ -363,6 +363,35 @@ <h3>Triple Terms and Reification</h3>
 
     <p>Note that a <a>triple term</a> may also have another <a>triple term</a> as an <a>object</a>.</p>
   </section>
+  
+  <section id="section-version-announcement">
+    <h3>RDF version announcement</h3>
+	
+    <p>To allow RDF parsers to error or warn about unsupported RDF versions as early as possible,
+		RDF serialization formats SHOULD allow a version to be specified,
+		either via the HTTP Content-Type header,
+		a version announcement in the format-specific syntax,
+		or both.</p>
+
+	<p>If the version is announced in both the Content-Type header and in the syntax, they MUST be equal.</p>
```

Comment:
This is an informative section, so we should avoid BCP14 words like SHOULD and MUST, or move them to a normative section.

Reply:
```suggestion
	<p>If the version is announced in both the Content-Type header and in the syntax, they are expected to be equal.</p>
```